scripts/docker-integration-tests/prom_remote_write_backend/utils.sh

```diff
@@ -0,0 +1,152 @@
+#!/usr/bin/env bash
+
+set -xe
+
+source "$M3_PATH"/scripts/docker-integration-tests/common.sh
+
+function prometheus_remote_write {
+  local metric_name=$1
+  local datapoint_timestamp=$2
+  local datapoint_value=$3
+  local expect_success=$4
+  local expect_success_err=$5
+  local expect_status=$6
+  local expect_status_err=$7
+
+  network_name="prom_remote_write_backend_backend"
+  network=$(docker network ls | grep -F $network_name | tr -s ' ' | cut -f 1 -d ' ' | tail -n 1)
+
+  out=$( (docker run -it --rm --network "$network"          \
+    "$PROMREMOTECLI_IMAGE"                                  \
+    -u http://m3coordinator01:7201/api/v1/prom/remote/write \
+    -t __name__:"${metric_name}"                            \
+    -d "${datapoint_timestamp}","${datapoint_value}" | grep -v promremotecli_log) || true)
+
+  success=$(echo "$out" | grep -v promremotecli_log | docker run --rm -i "$JQ_IMAGE" jq .success)
+  status=$(echo "$out" | grep -v promremotecli_log | docker run --rm -i "$JQ_IMAGE" jq .statusCode)
```

Comment:
Why are these being piped though jq docker container? Could it pipe through local jq instead (i.e.`... | jq ...`) like in other functions in this file?

Reply:
I am not sure. My guess - to not force installation on JQ on a running machine. I copied (and removed some uneeded stuff) from other tests.

Reply:
That seems reasonable. However, IIUC, other code in the same file uses local jq, e.g.

https://github.com/m3db/m3/blob/36e3845d11718947810455e60fb0a39f0f071317/scripts/docker-integration-tests/prom_remote_write_backend/utils.sh#L50-L51

So we still have to have jq on machine, with the additional complexity of running some of the jq commands through docker container.

Reply:
ah yeah, actually I used local myself in couple of commands without thinking. Will use local here as well.